src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Utils/MockUtils.cs

```diff
@@ -189,5 +192,16 @@ private static Mock<TConfiguration> PopulateMockConfiguration<TConfiguration>(Tu
 
             return configuration;
         }
+
+        public static void LoggerSetup<T>(Mock<ILogger<T>> logger, LogLevel logLevel, string logMessage)
```

Comment:
I would personally rename this to something like `SetupLoggerExpectation` or `ExpectLogMessage`.  `LoggerSetup` is quite a generic name that doesn't broadcast what its purpose is to me.

Reply:
Or you could make it an extension method of Mock<ILogger<T>> and then you could rewrite:

`MockUtils.LoggerSetup(logger, LogLevel.Warning, expectedWarning);`

as:

`logger.SetupExpectedMessage(LogLevel.Warning, expectedWarning)` or `logger.SetupExpectedWarning(expectedWarning)`